examples/arc.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+
+class ExampleArcApp(monome.ArcApp):
+    def __init__(self):
+        super().__init__()
+        self.pos = [0, 0, 0, 0]
+
+    def on_arc_ready(self):
+        print('Ready, clearing all rings...')
+        for n in range(0, 4):
+            self.arc.ring_all(n, 0)
+
+    def on_arc_disconnect(self):
+        print('Arc disconnected.')
+
+    def on_arc_delta(self, ring, delta):
+        print('Ring: {} Delta: {}'.format(ring, delta))
+
+        old_pos = self.pos[ring]
+        new_pos = old_pos + delta
+
+        if new_pos > old_pos:
+            for p in range(old_pos, new_pos):
+                self.arc.ring_set(ring, p, 15)
+        else:
+            for p in range(new_pos, old_pos):
+                self.arc.ring_set(ring, p, 0)
+        self.pos[ring] = new_pos
+
+    def on_arc_key(self, ring, s):
+        print('Ring: {} Pressed:{}'.format(ring, s > 0))
```

Comment:
string formatting here could be consistent with the rest of the example:
```suggestion
        print('Ring: {} Pressed: {}'.format(ring, s > 0))
```